pandas/core/reshape/tile.py

```diff
@@ -142,12 +142,17 @@ def cut(
         fixed set of values.
     Series : One-dimensional array with axis labels (including time series).
     IntervalIndex : Immutable Index implementing an ordered, sliceable set.
+    np.histogram_bin_edges: Function to calculate only the edges of the bins
```

Comment:
```suggestion
    numpy.histogram_bin_edges: Function to calculate only the edges of the bins
```

From the [docstring contribution docs](https://pandas.pydata.org/pandas-docs/stable/development/contributing_docstring.html#section-5-see-also):

> When relating to other libraries (mainly `numpy`), use the name of the module first (not an alias like `np`). If the function is in a module which is not the main one, like `scipy.sparse`, list the full module (e.g. `scipy.sparse.coo_matrix`).